app/controllers/tags_controller.rb

```diff
@@ -17,7 +17,7 @@ class TagsController < ApplicationController
   def index
     skip_authorization
     @tags_index = true
-    @tags = Tag.where(alias_for: [nil, ""]).includes(:sponsorship).order(hotness_score: :desc).limit(100)
+    @tags = Tag.concrete.includes(:sponsorship).order(hotness_score: :desc).limit(100)
```

Comment:
I'm not sure how much I like the name `concrete`. When I read your post on Slack the first term that came to mind was `direct`, as in a direct reference vs an indirect one like `rubyonrails` -> `rails`.

Reply:
Yes!  I like `direct` more.